src/components/AddUser/AddUser.component.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { Align, Button, ContainerInput, Forms, Input, Inputdata, Label, List } from "./AddUser.component.styled";
+
+
+export const AddUser = () =>{
```

Comment:
mesma coisa aqui, to achando q seu linter nao ta pegando

Reply:
vish